.gitignore

```diff
@@ -137,3 +137,5 @@ env
 # Matching pyproject.toml
 paperqa/version.py
 tests/example*
+tests/test_index/*
+.DS_Store
```

Comment:
We should just generate this file:

```
curl -s \
  https://raw.githubusercontent.com/github/gitignore/master/\{Global/Vim,Global/JetBrains,Global/VisualStudioCode,Global/macOS,Python\}.gitignore \
  > .gitignore
```

Then copy over the paper-qa specifics

Reply:
hh that's that this file already was, previously it just didn't include the VS code section. I added it back.